_policies/regressions.md

```diff
@@ -1,8 +1,39 @@
 ---
-title: Regressions
+title: How to Mark Regressions
 layout: policy
-published: false
+published: true
 description: Marking-up regressions
 ---
 
-Page under development.
\ No newline at end of file
+# What is a regression?
+
+A regression is a bug (in our scheme a `defect`) introduced by a change set. 
+
+- Bug 101 *fixes* Bug 100 with Change Set A
+- Bug 102 *reported which describes previously correct behavior now not happening*
+- Bug 102 *investigated and found to be introduced by Change Set A*
+
+# Marking a Regression Bug
+
+These things are true about regressions:
+
+- **Bug Type** is `defect`
+- **Keywords** include `regression`
+- **Status_FirefoxNN** is `affected` for each version (in current nightly, beta, and release) of Firefox in which the bug was found
+- The bug's description covers previously working behavior which is no longer working [ed. I need a better phrase for this]
+
+Until the change set which caused the regression has been found through [mozregression](https://mozilla.github.io/mozregression/) or another bisection tool, the bug should also have the `regressionwindow-wanted` keyword. 
+
+Once the change set which caused the regression has been identified, remove the `regressionwindow-wanted` keyword and set the **Regressed By** field to the id of the bug associated with the change set.
+
```

Comment:
We also need to NeedInfo the author of the regressing patch in the bug and ask him to fix the regression.

Reply:
Added